core/chainio/builder.go

```diff
@@ -0,0 +1,214 @@
+package chainio
+
+// Rewrite of 'github.com/Layr-Labs/eigensdk-go/chainio/clients/builder.go' to be more flexible
```

Comment:
Specifically, this version: https://github.com/Layr-Labs/eigensdk-go/blob/33241d37ea3a4d3ed3cfbb01010a3c70cad494d0/chainio/clients/builder.go#L1